### PR TITLE
Travis added

### PR DESCRIPTION
**Pull request by TomasVotruba**
This will run tests automatically against CI

Needs to be added here: https://travis-ci.org/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - 5.5
+  - 5.6
+  - 7.0
+
+install:
+  - composer install --prefer-source
+
+before_script:
+  - composer selfupdate
+  - composer update --prefer-dist
```
Comment (EmanueleMinotto) [inline .travis.yml]: this is redundant

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+script:
+  - phpunit
```
Comment (EmanueleMinotto) [inline .travis.yml]: default Travis script for php builds is `phpunit`([ref](https://docs.travis-ci.com/user/languages/php/#PHPUnit)), so imo could be removed

**Comment by TomasVotruba** [comment]
Closing in favor of #21

File: .travis.yml (hunk 2 above)
Comment (TomasVotruba) [inline .travis.yml]: I know, just prefer to be explicit. It seem like magic for those who don't know.

**Comment by EmanueleMinotto** [comment]
uhm... shouldn't this be splitted and merged before from the big PR? So that big PR can be tested with Travis before merging

**Comment by TomasVotruba** [comment]
I test it already in the fork. This package is not on Travis yet.

**Comment by EmanueleMinotto** [comment]
:+1:

**Pull request closed without merging**